ui/app/router.js

```diff
@@ -26,7 +26,9 @@ Router.map(function () {
       this.route('license');
       this.route('mfa-setup');
       this.route('clients', function () {
-        this.route('dashboard');
+        this.route('overview');
+        this.route('token');
```

Comment:
My initial thought was to name the route `entity` but was getting hung up on the fact that it encompasses both entity and non-entity clients. Token seemed fitting since it is counting authenticated users/services to Vault which requires a token.